.github/pull_request_template.md

```diff
@@ -38,6 +38,7 @@ Please provide a summary of the tests affected by this work and any unique strat
 ### General
 <!--- Review the list and put an x in the boxes that apply. -->
 
+- [ ] I have included a change request file using $ npm run change
```

Comment:
curious if this is needed. The CLI, at least on FAST, automatically would fail if the change log file is not provided and would error out on the terminal with a like message.

Does this repo work the same way? If so, it may not be needed as a check box item.

Reply:
I suggest adding a more robust ci-validate-pr.yml file as a workflow to run each time. You may reference https://github.com/microsoft/fast/blob/master/.github/workflows/ci-validate-pr.yml to see the `checkchange` call used to perform this test.

Reply:
I've only really copied and pasted workflow scripts before, and I'm not sure how to test them without committing them. Give this a good look please and see if it looks right for our setup.